### DISCUSSION TITLE
What happened to neofetch?

### DISCUSSION THREAD

**Discussion by IgorD-lab** [prasanthrangan/hyprdots#1524] [Q&A] [answered]
![image](https://github.com/prasanthrangan/hyprdots/assets/74680803/388f4d8b-bae1-4132-80b9-eec621cf8c24)

Just did a fresh install and noticed that old neofetch is gone what happened?

**Comment by kRHYME7** [comment, marked as answer]
we're using fastfetch now.

**Comment by IgorD-lab** [comment]
Oh alright is there a way to make terminal open with fastfetch instead of pokemon. 
There was another issue about this #494. Solution was to comment the pokemon line in .zshrc file and uncomment the neofetch line which has been removed if im not mistaken so im not sure how to proceed.

**Reply by kRHYME7** [reply]
remove the pokemon line and put ` fastfetch `
